modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/app/App.es.js

```diff
@@ -13,10 +13,10 @@
  */
 
 import {openToast} from 'frontend-js-web';
-import core from 'metal';
 import dom from 'metal-dom';
 
 import {App} from '../../senna/senna';
+import utils from '../../senna/utils/utils';
```

Comment:
`utils/utils` ... I know I brought this up in the prior import, so just mentioning it again so that we don't forget it — at some point we should clean up the paths/structure here to avoid this unsightly repetition.

A pretty bad smell in here, IMO, is that we are now importing `utils` from one file and `Utils` from another 🤦‍♂️. We should probably be importing named exports anyway (eg. in this instance, we only want `getUid()`, so we should import only that).

Reply:
As I mentioned offline, I already did this and it broke all tests in an irrecoverable way.

I spent some time trying to clean it up but it was impossible for me to figure out what type of shared state of race condition is at play in here, so had to defer this.

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/489#issuecomment-725325366